src/debugger/common/channel/MessageMarshaller.js

```diff
@@ -48,13 +48,20 @@ export class MessageMarshaller {
     return `${requestID} ${messageType} ${JSON.stringify(breakpoints)}`;
   }
 
-  marshallStoppedResponse(reason: StoppedReason, filePath: string, line: number, column: number): string {
+  marshallStoppedResponse(
+    reason: StoppedReason,
+    filePath: string,
+    line: number,
+    column: number,
+    message?: string
+  ): string {
     let result: StoppedResult = {
       kind: "stopped",
       reason: reason,
       filePath: filePath,
       line: line,
       column: column,
+      message: message || "",
```

Comment:
Instead of making this "" if not supplied, keep it undefined. JSON.stringify will then omit it from the string. That seems like a better way of handling an optional parameter.